src/frontend/js/types/Joanie.ts

```diff
@@ -102,7 +102,8 @@ export interface Certificate {
   id: string;
   issued_on: string;
   certificate_definition: CertificateDefinition;
-  order: NestedCertificateOrder | NestedCredentialOrder;
+  order: Nullable<NestedCertificateOrder | NestedCredentialOrder>;
```

Comment:
WDYT about trying to make an exclusion type like "if `order` is defined then `enrollment` is not" and vice-versa? I'm not saying that's easy though 😅

Reply:
Please review the fixup commit to check if the proposed solution is fine for you.